src/ConfiguratorServer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\RoadRunner\Http;
+
+use function time;
+use function microtime;
+
+class ConfiguratorServer
+{
+    /**
+     * Returns altered copy of _SERVER variable. Sets ip-address,
+     * request-time and other values.
+     *
+     * @return non-empty-array<array-key|string, mixed|string>
+     */
+    public function configure(Request $request): array
```

Comment:
It's better to make this method static
```suggestion
    public static function populateServer(Request $request): void
```

Reply:
Fixed